AutoRest/Generators/NodeJS/NodeJS.Tests/Expected/AcceptanceTests/BodyFile/operations/files.js

```diff
@@ -82,7 +82,7 @@ Files.prototype.getFile = function (options, callback) {
   httpRequest.body = null;
   // Send Request
   httpRequest.streamedResponse = true;
-  return client.pipeline(httpRequest, function (err, response) {
+  return client.pipeline(httpRequest, function (err, response, responseBody) {
```

Comment:
the third parameter will mostly be undefined for streaming scenario, unless an erroneous response is sent (4xx, 5xx) by the server.